pages/api/openPR.js

```diff
@@ -277,14 +278,25 @@ export default async (req, res) => {
               }
             );
           }
+        } else {
+          // push error object into results array
+          pullRequestNumbers.push(result);
+          break;
         }
-      });
+      }
 
       return pullRequestNumbers;
     });
 
-    // return an unconditional success response
-    res.statusCode = 200;
```

Comment:
This is my code, yet, this idea of unconditionally returning `success` doesn't make sense anymore. And I can't figure out how the errors were passed before, so I refactored it to properly pass the errors.